pkg/provider/aws/meta/details.go

```diff
@@ -29,7 +29,7 @@ import (
 
 func WithDefaultEC2() Option {
 	return func(o *options) error {
-		o.ec2Type = []string{"c5*", "t3*", "m5*", "r5*"}
+		o.ec2Type = []string{"c5*", "t3*", "m5*", "r5*", "m7i*", "m7a*", "m8g*"}
```

Comment:
for now we are not showing users option to choose arm64 based instances as we have have further modifications left in the `instance_image_getter` so that it handles for both x86-64 based image as well as arm64 based image

Reply:
context of this message is the availability of instancetypes `m8g.*` as these are aws graviton instances